registration.md

```diff
@@ -4,6 +4,8 @@ title: Registration
 ---
 A `Registration` object describes one school or other organization's entry into a tournament. It comprises the basic details about an organization and one or more [`Team`][team] objects.
 
+It is expected that each `Registration` object represents a distinct school or other organization. For instance, if "Anywhere High School" enters two teams, there should not be two `Registration` objects whose `name` fields are `Anywhere A` and `Anywhere B` and one `Team` child each, there should be one `Registration` object whose `name` field is `Anywhere` (or `Anywhere High School`) and two `Team` children.
```

Comment:
I think this is correct as written (compare to "Don't go to Tarshish, go to Nineveh") but I'm not completely sure (I don't know grammatical rules, I just go by what sounds right to me, and am nearly always correct) and I'm fine with semicoloning it. Coming right up…